www/src/scss/styles.scss

```diff
@@ -401,10 +401,10 @@ a {
   }
```

Comment:
The styling for the insert pane looks off
![image](https://user-images.githubusercontent.com/2982784/104134240-3f987a80-533d-11eb-9892-89a94c8b9cb3.png)

Reply:
We should add a min width to the textbox
![image](https://user-images.githubusercontent.com/2982784/104536648-43dac700-55cd-11eb-926e-f58fee4c9864.png)